src/version.js

```diff
@@ -1 +1 @@
-export default '6.0.8-ded3';
+export default '6.0.8-ded4';
```

Comment:
I wonder why the version is duplicated in the package json and here 🤔

Reply:
This one is not use at all, in the official package they migrated to use `semantic-release`. So I guess this file is a left over of that migration. My guess is that it wasn't removed since it was exposed through the index.js.  ( I don't know if somehow they replace this value when releasing)

Reply:
alright maybe can be deleted? @aminahasimbegovic

Reply:
not sure, if it's exposed as Felipe said